server/lib/dispatch.js

```diff
@@ -17,4 +18,17 @@ export async function dispatch({ address }) {
     ]
   })
   return { address, transaction: receipt.id }
-}
\ No newline at end of file
+}
+
+export async function dispatchToAoVouchDao({ address }) {
```

Comment:
Consider naming `send` or `message` or `sendMessage` - don't think we need to specify VouchDAO or AO, seems to me the project purpose successfully implies both.

Reply:
Changed to `sendMessage()`